CI/codefresh.yml

```diff
@@ -55,11 +55,122 @@ steps:
     type: build
     image_name: soluto/e2e-integration
     working_directory: ./e2e/integration
+  
+  BuildingApiSmokeTestImage:
+    title: Building Tweek Api Smoke Test Docker Image
+    type: build
+    image_name: soluto/tweek-api-smoke-tests
+    working_directory: ${{main_clone}}
+    dockerfile: TweekApiSmokeTestDockerfile
 
   Tests:
     title: Tests - e2e & smoke
     type: composition
-    composition: tweek_test_composition
+    composition:
```

Comment:
Please, create the composition on codefresh. You can name it `tweek_test_coomposition_zap`.

Reply:
Actually, I discussed this with @nataly87s and we both think this is better - she even took it one step forward and use the same `docker-compose.yml` file.